src/desktop_notifier/main.py

```diff
@@ -294,3 +295,17 @@ async def get_capabilities(self) -> frozenset[Capability]:
         if not self._capabilities:
             self._capabilities = await self._impl.get_capabilities()
         return self._capabilities
+
+    async def count_delivered_notifications(self) -> int:
+        """
+        Returns the number of notifications ever sent for this app.
+        """
+        from .macos import CocoaNotificationCenter
+        from .macos_support import macos_version
```

Comment:
Those imports will likely fail on other platforms. Instead, the idiomatic approach would be to add `count_delivered_notifications` to the abstract base class `base.DesktopNotifierBase` and let the different implementation decide if they support it or not.

FYI, this ImportError does not currently show up in tests because you haven't added any for this method :P

Reply:
yeah it's marked `[WIP]` bc i wasn't sure it was even worth adding; figured i'd spend the extra work on tests etc. if it turned out to be worth it.